scout/utils/convert.py

```diff
@@ -53,7 +53,7 @@ def amino_acid_residue_change_3_to_1(protein_sequence_name):
     if protein_sequence_name is None:
         return None
 
-    p = re.compile("p.([A-Za-z]+)(\d+)([A-Za-z]+)")
+    p = re.compile(r"p.([A-Za-z]+)(\d+)([A-Za-z]+)")
```

Comment:
```suggestion
    p = re.compile(r"p\.([A-Za-z]+)(\d+)([A-Za-z]+)")
```

Asked ChatGPT for this one

Reply:
Hehe, it is right, although the other version kind of works as well (it should find just one of any char, which should almost always be a literal `.` given the rest of the re.